cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ReplicationThrottleHelper.java

```diff
@@ -189,8 +198,26 @@ private void setThrottledReplicas(String topic, Set<String> replicas, boolean th
     if (currThrottledReplicas != null) {
       newThrottledReplicas.addAll(Arrays.asList(currThrottledReplicas.split(",")));
     }
-    config.setProperty(replicaThrottleConfigKey, String.join(",", newThrottledReplicas));
-    ExecutorUtils.changeTopicConfig(_adminZkClient, topic, config);
+    topicConfigs.setProperty(replicaThrottleConfigKey, String.join(",", newThrottledReplicas));
+    changeTopicConfigs(topic, topicConfigs);
+  }
+
+  /**
+   * Update topic configuration properties if a topic exist. No-op for non-existent topic
+   *
+   * @param topicName topic name
+   * @param topicConfigs configuration properties to update
+   */
+  private void changeTopicConfigs(String topicName, Properties topicConfigs) {
+    try {
+      ExecutorUtils.changeTopicConfig(_adminZkClient, topicName, topicConfigs);
+    } catch (Exception e) {
```

Comment:
The `AdminZkClient.changeTopicConfig` API does not claim that it might throw `AdminOperationException`. Even though we know that is the kind of exception it might throw, we should not make this assumption unless it's part of the public contract.